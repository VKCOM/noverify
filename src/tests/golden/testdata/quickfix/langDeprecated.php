<?php

define("CONST", 10, false);
define("CONST1", 10, true);
define("CONST2", 10);
