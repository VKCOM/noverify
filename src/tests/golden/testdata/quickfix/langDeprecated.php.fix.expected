<?php

define("CONST", 10);
define("CONST1", 10, true);
define("CONST2", 10);
